modules/solid_mechanics/include/postprocessors/StepNumber.h

```diff
@@ -0,0 +1,33 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "GeneralPostprocessor.h"
+#include "StepUserObject.h"
+
+/**
+ * Given the current time, outputs the current analysis step number
+ */
+class StepNumber : public GeneralPostprocessor
```

Comment:
It would be nice to have a longer name. StepNumber will show up in the general index and confuse user